gravitee-am-model/src/main/java/io/gravitee/am/model/User.java

```diff
@@ -771,4 +771,15 @@ public void setAddress(Map<String, Object> address) {
         putAdditionalInformation(StandardClaims.ADDRESS, address);
     }
 
+    public void updateUsername(String username) {
+        setUsername(username);
+        setLastUsernameReset(new Date());
+        unlockUser();
```

Comment:
Shouldn't we have a `setUpdatedAt` ?
Or this is done by the underlying `getUserService()` method ? (both domain and organization)

Reply:
Done by the underlying `getUserService()` method.

https://github.com/gravitee-io/gravitee-access-management/blob/fd7f78ee47d337420320456912ff31d16b217fd4/gravitee-am-service/src/main/java/io/gravitee/am/service/impl/OrganizationUserServiceImpl.java#L114

https://github.com/gravitee-io/gravitee-access-management/blob/fd7f78ee47d337420320456912ff31d16b217fd4/gravitee-am-service/src/main/java/io/gravitee/am/service/impl/UserServiceImpl.java#L134